packages/at_client/test/notification_service_test.dart

```diff
@@ -162,6 +162,7 @@ void main() {
               AtClientPreference()..namespace = 'wavi',
               mockSharedKeyEncryptionImpl)
           .transform(notificationParams);
+      //UPPER_CASE in the forText() method will be preserved
```

Comment:
Clarify the comment - maybe, "Upper case should be preserved when using forText notifications"